src/Core/Checkout/Customer/Event/AddressHashEvent.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Checkout\Customer\Event;
+
+use Shopware\Core\Checkout\Customer\Aggregate\CustomerAddress\AddressHashStruct;
+use Shopware\Core\Framework\Log\Package;
+use Symfony\Contracts\EventDispatcher\Event;
+
+#[Package('checkout')]
+class AddressHashEvent extends Event
```

Comment:
Why all of this?

Why not simply create a custom subscriber. If a plugin wants to extend on that, it can listen to the same address loaded events and simply modify the hash, however it wants :shrug: